lib/Normalize_std_ast.ml

```diff
@@ -392,20 +408,34 @@ let make_mapper conf ~ignore_doc_comments ~erase_jane_syntax =
     Ast_mapper.default_mapper.typ m typ
   in
   let structure =
+    let structure m str =
+      List.filter str ~f:(fun stri ->
+          match Jane_syntax.Structure_item.of_ast stri with
+          | Some (Jstr_layout (Lstr_kind_abbrev _)) -> false
+          | _ -> true )
+      |> Ast_mapper.default_mapper.structure m
+    in
```

Comment:
This, and the similar thing just below, seem surprising to me.  Do I understand correctly we're just filtering out kind abbrevs entirely before doing the round-trip check?

Reply:
You're right, we should only be filtering them when we're erasing jane syntax (fixed)